### PR TITLE
Add liblas and pymcc_lidar

### PR DESCRIPTION
**Pull request by rmsare**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rmsare** [comment]
This package is developed for Linux/OS X, so Windows CI builds will probably fail.

**Comment by rmsare** [comment]
This package depends on LibLAS, which is only available from the `auto` channel and so can't be used with `conda build`. It's a C dependency.

I may try to repackage liblas in `conda-forge`, which seems to be the only solution unless conda/conda-build#532 is fixed.

Would this be the recommended way to fix this build @conda-forge/help-python?

**Comment by isuruf** [comment]
@conda-forge, we only use packages from defaults and conda-forge. You should add `liblas` as a conda package to conda-forge.

**Comment by rmsare** [comment]
@isuruf I've created a LibLAS recipe for conda-forge. 

Can I add it in this PR? Or should I close this and add it in a separate PR, then resubmit?

**Comment by isuruf** [comment]
You can add it to this PR.

File: recipes/pymccrgb/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+python setup.py --quiet install --single-version-externally-managed --record=record.txt
+pip install git+https://github.com/stgl/pymcc
```
Comment (isuruf) [inline recipes/pymccrgb/build.sh]: Why add this here?

Comment (rmsare) [reply recipes/pymccrgb/build.sh]: This is a dependency of the pymccrgb package not on pypi. It was originally installed from a git repo as part of the environment.

Comment (rmsare) [reply recipes/pymccrgb/build.sh]: Is it required to upload pymcc to pypi, add a conda recipe, and install it via conda-forge channel?

Comment (isuruf) [reply recipes/pymccrgb/build.sh]: Yes.

**Comment by rmsare** [comment]
@isuruf Most of the issues have been solved. But in the latest build, the `pymccrgb` dependencies show [a lot of conflicts](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=84107&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=7636&lineEnd=7636&colStart=1&colEnd=13) arising from `zstd`. 

Any idea on how to fix this without pinning all these dependencies?

**Comment by rmsare** [comment]
Nevermind, that is resolved after cleaning up host deps. 

I think the next linux build will pass, after I added a missing dependency.

**Comment by rmsare** [comment]
The liblas and pymcc_lidar recipes pass, but there are a lot of unsatisfiable dependencies for the main recipe. I'm not sure why this is, as the only new dependencies are pdal/scipy. 

@isuruf Do you have any idea why this is happening, and what might be a good fix?

**Comment by rmsare** [comment]
@gehilley You're listed as a maintainer for pymcc_lidar package. Could you comment here to confirm this?

@hobu I've listed you as a maintainer in the liblas recipe. Could you comment here to confirm this?

**Comment by gehilley** [comment]
Yes

**Comment by hobu** [comment]
no problem acting as a liblas maintainer

File: recipes/pymcc_lidar/build.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export PYTHON_EXECUTABLE=$CONDA_PYTHON_EXE
```
Comment (isuruf) [inline recipes/pymcc_lidar/build.sh]:
```suggestion
export PYTHON_EXECUTABLE=$PYTHON
```

**Comment by rmsare** [comment]
@isuruf Do you have any thoughts on the CI failure here? 

The libLAS and pymcc_lidar recipes pass, but a large number of unsatisfiable dependencies arise with pymccrgb.

I have tried pinning one of the problematic dependencies (numpy) which did not help. I also removed fiona, which does require GDAL < 3.0.

**Comment by rmsare** [comment]
I am going to handle the pymccrgb recipe in a separate PR after I fix it locally. 

Sorry for my delay with this, but I've removed the failing recipe. This PR will only add the two other packages which pass CI.

**Comment by rmsare** [comment]
@isuruf I think these recipes are good to go. Do you think this could be merged? Thanks.

**Comment by hobu** [comment]
This is ready to merge. There is also a libLAS release in the works that will catch up a number of maintenance and bug issues that have accumulated over the last 4 years.

**Pull request closed without merging**